helm/net-exporter-chart/templates/rbac.yaml

```diff
@@ -56,6 +56,4 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: net-exporter
-  labels:
-    app: net-exporter
```

Comment:
This is incorrect but Helm 2 doesn't complain. Helm 3 is stricter and rejects this.